dlang/source/flatbuffers/flatbufferbuilder.d

```diff
@@ -0,0 +1,301 @@
+module flatbuffers.flatbufferbuilder;
+
+import flatbuffers.exception;
+import flatbuffers.bytebuffer;
+
+import std.exception;
+
+final class FlatBufferBuilder
+{
+public: 
+	this(int initsize)
+	{
+		if(initsize == 0)
+			throw new ArgumentOutOfRangeException("initsize", initsize, "Must be greater than zero");
```

Comment:
Either this should give you a message `"must not be zero"` or check for `<= 0`

The Java implementation just sets it to 1 instead of throwing an exception